src/components/organisms/userPosts/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { api } from "~/utils/api"
+import MoleculesPost from "~/components/molecules/post";
+
+type UserPostsProps = {
+  userId: string
+}
+
+const UserPosts = ({ userId }: UserPostsProps) => {
```

Comment:
```suggestion
const OrganismsUserPosts = ({ userId }: OrganismsUserPostsProps) => {
```
`Organisms`をつけていきましょう！